netutils/interface.py

```diff
@@ -119,15 +122,21 @@ def canonical_interface_name(interface, addl_name_map=None, verify=False):
         name_map.update(addl_name_map)
     # check in dict for mapping
     if name_map.get(interface_type):
-        long_int = name_map.get(interface_type)
+        long_int = name_map.get(interface_type, "")
```

Comment:
This should just be a `name_map[interface_type]` since it is checked if it exists already

Reply:
Fixed!